projects/packages/jetpack-mu-wpcom/src/features/wpcom-media/wpcom-media-url-upload-form/style.scss

```diff
@@ -0,0 +1,19 @@
+.wpcom-media-url-upload-form {
+	button#{&}__pre-upload-button {
+		padding: 0 21px;
+		margin-top: -15px;
+		margin-bottom: 20px;
+	}
+
+	input {
+		width: 100%;
+		max-width: 600px;
+		margin-right: 6px;
+		margin-bottom: 5px;
+	}
+
+	form {
+		margin-top: -15px;
+	    margin-bottom: 21.5px;
```

Comment:
nit: the style looks weird.

Reply:
Yeah, but it's the only thing that works 😂

Reply:
Sorry I misunderstood your comment. Fixed, also simplified the styles a bit.